supersuit/vector/constructors.py

```diff
@@ -18,7 +18,7 @@ def MakeCPUAsyncConstructor(max_num_cpus):
 
         def constructor(env_fn_list, obs_space, act_space):
             example_env = env_fn_list[0]()
-            envs_per_env = getattr(example_env, "num_envs", 1)
+            total_envs = sum([getattr(env_fn(), "num_envs", 1) for env_fn in env_fn_list])
```

Comment:
Some environments are quite expensive to construct, seems kind of uncomfortable to contruct all of them by default. See discord for a solution where we just rely on users to compute this number when they need it.

Reply:
This makes sense, I'll close this PR then.